S2VX.Game/Editor/Containers/EditorInfoBar.cs

```diff
@@ -12,41 +12,38 @@ public class EditorInfoBar : CompositeDrawable {
         private EditorScreen Editor { get; set; }
 
         private ToolDisplay ToolDisplay { get; } = new ToolDisplay {
-            RelativeSizeAxes = Axes.Both,
-            RelativePositionAxes = Axes.Both,
             Anchor = Anchor.TopLeft,
             Origin = Anchor.TopLeft,
-            Width = 0.4f,
+            Width = 400,
             TextAnchor = Anchor.TopLeft,
         };
+        public ApproachRateDisplay ApproachRateDisplay { get; } = new ApproachRateDisplay {
```

Comment:
Use `new()` syntax

Reply:
I don't think I can here. When I try to I get [CS0029](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0029?f1url=%3FappId%3Droslyn%26k%3Dk(CS0029))

Reply:
This works for me

```suggestion
        public ApproachRateDisplay ApproachRateDisplay { get; } = new() {
            Anchor = Anchor.TopLeft,
            Origin = Anchor.TopLeft,
            X = 365,
            Width = 200,
            TextAnchor = Anchor.TopRight,
        };
```

Reply:
Oh I forgot to put the parentheses Lol. Fixed

Reply:
Can you go through and fix other `new()` cases? For example, there are some `new S2VXStory()` that could be simplified. 

It's kind of a pain to track down all of these, so  I think I'll probably stop commenting about `new()` in general after this. It's good to be consistent, but this syntax change is not something I want us spending a lot of time on hunting down. This kind of thing should be handled by automatic code analysis.

Reply:
Aite, I went through all the files I touched in this PR and changed it all to new() where it's possible. Yeah it's kind of a pain to keep track of every use but I guess we should just keep it in mind whenever we initialize a new property/variable. It would be a lot more convenient if there was a automatic check for it though

Reply:
Yeah, unfortunately idk if a check like this will ever come. We talked about this before, but there's probably some benefits to generalize your types as much as possible, e.g. take an `IEnumerable` instead of a `List`. This way you can slot things in with less hassle. So if the code analysis rules were to uphold that, enforcing `new()` would be awkward.